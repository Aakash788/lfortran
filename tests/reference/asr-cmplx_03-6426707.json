{
    "basename": "asr-cmplx_03-6426707",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/cmplx_03.f90",
    "infile_hash": "20d38686e1c0b08e83731b13995ea390ac379958c2eaea9a03bebf87",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-cmplx_03-6426707.stderr",
    "stderr_hash": "b4ec448100aff185a1210c311a330fc865fb1865aae64efde5d1293a",
    "returncode": 2
}