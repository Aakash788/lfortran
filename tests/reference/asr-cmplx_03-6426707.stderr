semantic error: Syntax error in `complex` constant
 --> tests/errors/cmplx_03.f90:2:13
  |
2 |     print*, cmplx((1.00000000, real(0, kind=4)), kind=8)
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
